### ISSUE TITLE
Why are we restricting just to the first static code per patient? Shouldn't we want to summarize all static codes?

### ISSUE THREAD

**Issue by mmcdermott** [mmcdermott/MEDS_Tabular_AutoML#76]
https://github.com/mmcdermott/MEDS_Tabular_AutoML/blob/48550d16664738f375abb4ba7cfff21f79dcbb07/src/MEDS_tabular_automl/generate_static_features.py#L120

@Oufattole and @teyaberg , am I just misunderstanding something here?

**Comment by mmcdermott** [comment]
We should add doctests to this function either way.

**Comment by mmcdermott** [comment]
Maybe did you mean to group by both "patient_id" and "code"?

**Comment by Oufattole** [comment]
Yeah that's a bug, I meant to group by both "patient_id" and "code", great catch!

**Comment by mmcdermott** [comment]
@Oufattole is this solved?
